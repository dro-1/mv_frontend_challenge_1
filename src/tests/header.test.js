import React from 'react'
import { shallow } from 'enzyme'
import Header from './../components/header.component'

describe('Header Component', () => {
  it('expect to render Header', () => {
    expect(shallow(<Header name="Seun" />)).toMatchSnapshot()
  })
})
