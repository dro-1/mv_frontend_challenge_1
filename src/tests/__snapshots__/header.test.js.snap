// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component expect to render Header 1`] = `
<styled.nav>
  <header>
    <h1>
      Hi 
      Seun
    </h1>
  </header>
  <img
    alt="profile"
    src="https://cdn.pixabay.com/photo/2016/03/31/19/58/avatar-1295430_960_720.png"
  />
  <button>
     Log Out
  </button>
</styled.nav>
`;
